src/types/orcid.types.ts

```diff
@@ -5,3 +5,14 @@ export interface OrcidResponse {
   family_name: string | null;
   given_name: string | null;
 }
+
+export interface OrcidCredentials {
+  orcid: string | null;
```

Comment:
Hvis noen av disse feltene mangler så vil man få badRequest, men jeg tenker at det er greit nok at man får feilen fra api-kallet.
Hva tror dere?

Reply:
Hvis backend forventer en verdi for alle felter syns jeg typen burde gjenspeile det også, så hindres vi fra å sende ting vi vet vil feile :)

Reply:
Fikset